server.go

```diff
@@ -64,11 +69,11 @@ func (h *handler) GetChangesFromYAML(p *params.ChangesFromYAMLParams) (params.Ch
 func getChanges(bundleYAML string) ([]params.Change, error) {
 	bundle, err := charm.ReadBundleData(strings.NewReader(bundleYAML))
 	if err != nil {
-		return nil, fmt.Errorf("error reading bundle data: %v", err)
+		return nil, errgo.WithCausef(err, params.ErrUnparsable, "error reading bundle data")
 	}
 	err = bundle.Verify(nil, nil)
 	if err != nil {
-		return nil, fmt.Errorf("error verifying bundle data: %v", err)
+		return nil, errgo.WithCausef(err, params.ErrUnparsable, "error verifying bundle data")
```

Comment:
Is "unparsable" really the right code here? I'm wondering if you want an ErrVerificationFailure as well.